modules/calib3d/misc/java/test/Calib3dTest.java

```diff
@@ -237,22 +236,23 @@ public void testFindCirclesGridMatSizeMatInt() {
     }
 
     public void testFindFundamentalMatListOfPointListOfPoint() {
-        int minFundamentalMatPoints = 8;
-
-        MatOfPoint2f pts = new MatOfPoint2f();
-        pts.alloc(minFundamentalMatPoints);
-
-        for (int i = 0; i < minFundamentalMatPoints; i++) {
-            double x = Math.random() * 100 - 50;
-            double y = Math.random() * 100 - 50;
-            pts.put(i, 0, x, y); //add(new Point(x, y));
-        }
-
-        Mat fm = Calib3d.findFundamentalMat(pts, pts);
-
-        truth = new Mat(3, 3, CvType.CV_64F);
-        truth.put(0, 0, 0, -0.577, 0.288, 0.577, 0, 0.288, -0.288, -0.288, 0);
-        assertMatEqual(truth, fm, EPS);
+        fail("Not yet implemented");
```

Comment:
This behavior looks strange to me: "fail" should fail test instead of passing or skipping them.

BTW, Could we use "block" comments to minimize code changes (add `/* ... */` on new lines before and after test body)?